### DISCUSSION TITLE
How to deploy a load balanced web service with domain but no https redirect?

### DISCUSSION THREAD

**Discussion by AlexanderMair** [aws/copilot-cli#3060] [Q&A]
The question is simple, but i could not find anything regarding this. Is there a way to deploy a load balanced web service using something like `copilot app init --domain foo.bar` but with no https redirect?

**Comment by huanjani** [comment]
Hello, @AlexanderMair! 
Currently, yes-- all domains are redirected to HTTPS. We're curious what your use case is for no redirect, if you're willing to share.
Thanks!

**Reply by AlexanderMair** [reply]
Hello, @huanjani !
yes, of course. It was just for test purposes. In my web service i was connecting to a web-socket with no SSL, but modern browsers do not allow a connection to an unencrypted web-socket from a https page. It may be an idea to add an option for no redirect in a test environment.
Thanks!

**Reply by webjunkie** [reply]
My use case would be that it's an API only service that all clients should connect to using HTTPS directly. No need to listen to port 80 anywhere. In addition, I saw some pots probing paths on port 80 which is not nice.
